### PR TITLE
OCPCLOUD-1737: cluster-control-plane-machine-set: configure presubmits and periodics for 4.12/4.13

### PR DESCRIPTION
**Pull request by damdo**
This PR configures:
- e2e presubmits 4.12
- e2e periodics for 4.12 and 4.13

### CODE REVIEW

File: ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.12.yaml [hunk 1]
```diff
@@ -99,6 +105,53 @@ tests:
...
+    test:
+    - as: test
+      commands: |
+        make e2e-periodic
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+      timeout: 3h0m0s
+    workflow: ipi-aws
```
Comment (JoelSpeed) [inline ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.12.yaml:122]: This is a presubmit so not sure we need this one backported? We should only be testing/making changes to the E2E on the main branch right?

Comment (damdo) [reply ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.12.yaml:122]: We had a chat and decided to keep this to signal to inform us whether periodics can be successful at presubmit stage.

**Comment by JoelSpeed** [comment]
/pj-rehearse ack

We know these tests will fail initially, but we want to be able to test the E2E backport, so need to have this merged before the E2Es merge

**Pull request merged**